sources/packages/web/src/components/common/sin/AddExpiryDate.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <v-form ref="addExpiryDateForm">
+    <modal-dialog-base
+      title="Add expiry date"
+      :showDialog="showDialog"
+      max-width="730"
+    >
+      <template #content>
+        <error-summary :errors="addExpiryDateForm.errors" />
+        <div class="pb-2">
+          <span class="label-value"
+            >Enter a date that the SIN will expiry on. Please note you will not
+            be able to change this.</span
+          >
+        </div>
+        <!-- TODO Date picker is not available in the vuetify 3 version, so temporary usage of textfield and regex-->
+        <v-text-field
+          label="Expiry date"
+          class="mt-2"
+          placeholder="yyyy-MM-dd"
+          v-model="formModel.expiryDate"
+          variant="outlined"
+          :rules="[
+            (v) =>
+              v.match(/^\d{4}-\d{2}-\d{2}$/) ||
+              'Expiry end date is not in right format',
+          ]" />
+        <v-textarea
+          hide-details
```

Comment:
do we need `hide-details` here?

Reply:
confirmed with @hellolynn-tbtb to have hide-details="auto", which has both the worlds of not showing the space and when validation is hit, showing the error.